go.mod

```diff
@@ -5,31 +5,31 @@ go 1.21
 require (
 	github.com/MakeNowJust/heredoc v1.0.0
 	github.com/google/btree v1.1.2
-	github.com/google/go-cmp v0.5.9
+	github.com/google/go-cmp v0.6.0
 	github.com/google/gofuzz v1.2.0
 	github.com/google/uuid v1.6.0
 	github.com/jteeuwen/go-bindata v3.0.8-0.20151023091102-a0ff2567cfb7+incompatible
-	github.com/openshift/api v0.0.0-20231101013329-0d0d46454bb7
-	github.com/openshift/apiserver-library-go v0.0.0-20230621110634-a99412818848
-	github.com/openshift/build-machinery-go v0.0.0-20230306181456-d321ffa04533
-	github.com/openshift/client-go v0.0.0-20230607134213-3cd0021bbee3
-	github.com/openshift/library-go v0.0.0-20230808150704-ce4395c85e8c
+	github.com/openshift/api v0.0.0-20240410131754-6e4793fb6a4b
+	github.com/openshift/apiserver-library-go v0.0.0-20240313131158-facc40cc7688
+	github.com/openshift/build-machinery-go v0.0.0-20231128094528-1e9b1b0595c8
+	github.com/openshift/client-go v0.0.0-20240408153607-64bd6feb83ae
+	github.com/openshift/library-go v0.0.0-20240408090355-0b298992f8cc
 	github.com/spf13/cobra v1.8.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.8.4
-	go.opentelemetry.io/otel/trace v1.10.0
-	k8s.io/api v0.28.3
-	k8s.io/apiextensions-apiserver v0.28.3
-	k8s.io/apimachinery v0.28.3
-	k8s.io/apiserver v0.28.3
-	k8s.io/client-go v0.28.3
-	k8s.io/code-generator v0.28.3
-	k8s.io/component-base v0.28.3
-	k8s.io/klog/v2 v2.100.1
-	k8s.io/kube-aggregator v0.28.3
-	k8s.io/kube-openapi v0.0.0-20230717233707-2695361300d9
-	k8s.io/kubernetes v1.28.3
-	k8s.io/utils v0.0.0-20230505201702-9f6742963106
+	go.opentelemetry.io/otel/trace v1.19.0
+	k8s.io/api v0.29.2
+	k8s.io/apiextensions-apiserver v0.29.2
+	k8s.io/apimachinery v0.29.2
+	k8s.io/apiserver v0.29.2
+	k8s.io/client-go v0.29.2
+	k8s.io/code-generator v0.29.2
+	k8s.io/component-base v0.29.2
+	k8s.io/klog/v2 v2.120.1
+	k8s.io/kube-aggregator v0.29.2
+	k8s.io/kube-openapi v0.0.0-20231010175941-2dd684a91f00
+	k8s.io/kubernetes v1.29.2
+	k8s.io/utils v0.0.0-20240310230437-4693a0247e57
```

Comment:
kube is on `v0.0.0-20230726121419-3b25d923346b` - https://github.com/kubernetes/kubernetes/blob/release-1.29/go.mod#L129C15-L129C49